Sources/MessagingInApp/State/Core/StoreSubscriber.swift

```diff
@@ -0,0 +1,28 @@
+// swiftlint:disable:next file_header
+//
+//  StoreSubscriber.swift
+//  ReSwift
+//
+//  Created by Benjamin Encz on 12/14/15.
+//  Copyright © 2015 ReSwift Community. All rights reserved.
+//
+
+public protocol AnyStoreSubscriber: AnyObject {
+    // swiftlint:disable:next identifier_name
+    func _newState(state: Any)
+}
+
+public protocol StoreSubscriber: AnyStoreSubscriber {
```

Comment:
this might not be needed as we creating the same thing in alias.

Reply:
To keep store generic and copy tests, it will be needed